src/Ztm.Data.Entity/Contexts/MainDatabase.cs

```diff
@@ -23,144 +24,123 @@ public MainDatabase(DbContextOptions<MainDatabase> options) : base(options)
         {
-            modelBuilder.Entity<Transaction>(b =>
-            {
-                b.Property(e => e.Hash).IsRequired();
-                b.Property(e => e.Version).IsRequired();
-                b.Property(e => e.LockTime).IsRequired();
-
-                b.HasKey(e => e.Hash);
-            });
+            builder.Property(e => e.Hash).IsRequired();
+            builder.Property(e => e.Version).IsRequired();
+            builder.Property(e => e.LockTime).IsRequired();
+
+            builder.HasKey(e => e.Hash);
         }
 
-        protected virtual void ConfigureWebApiCallback(ModelBuilder modelBuilder)
+        protected virtual void ConfigureWebApiCallback(EntityTypeBuilder<WebApiCallback> builder)
         {
-            modelBuilder.Entity<WebApiCallback>(b =>
-            {
-                b.Property(e => e.Id).IsRequired().ValueGeneratedNever();
-                b.Property(e => e.RegisteredIp).IsRequired();
-                b.Property(e => e.RegisteredTime).IsRequired();
-                b.Property(e => e.Completed).IsRequired();
-                b.Property(e => e.Url).IsRequired();
-
-                b.HasKey(e => e.Id);
-            });
+            builder.Property(e => e.Id).IsRequired().ValueGeneratedNever();
+            builder.Property(e => e.RegisteredIp).IsRequired();
+            builder.Property(e => e.RegisteredTime).IsRequired();
+            builder.Property(e => e.Completed).IsRequired();
+            builder.Property(e => e.Url).IsRequired();
+
+            builder.HasKey(e => e.Id);
         }
 
-        protected virtual void ConfigureWebApiCallbackHistory(ModelBuilder modelBuilder)
+        protected virtual void ConfigureWebApiCallbackHistory(EntityTypeBuilder<WebApiCallbackHistory> builder)
         {
-            modelBuilder.Entity<WebApiCallbackHistory>(b =>
-            {
-                b.Property(e => e.Id).IsRequired().ValueGeneratedOnAdd();
-                b.Property(e => e.CallbackId).IsRequired();
-                b.Property(e => e.Status).IsRequired();
-                b.Property(e => e.InvokedTime).IsRequired();
-                b.Property(e => e.Data).IsRequired();
-
-                b.HasKey(e => e.Id);
-                b.HasIndex(e => e.CallbackId); // Intentionally add
-                b.HasOne(e => e.Callback)
-                 .WithMany(e => e.InvocationHistories)
-                 .HasForeignKey(e => e.CallbackId)
-                 .HasPrincipalKey(e => e.Id)
-                 .OnDelete(DeleteBehavior.Cascade);
-            });
+            builder.Property(e => e.Id).IsRequired().ValueGeneratedOnAdd();
+            builder.Property(e => e.CallbackId).IsRequired();
+            builder.Property(e => e.Status).IsRequired();
+            builder.Property(e => e.InvokedTime).IsRequired();
+            builder.Property(e => e.Data).IsRequired();
+
+            builder.HasKey(e => e.Id);
+            builder.HasIndex(e => e.CallbackId); // Intentionally add
```

Comment:
Why do we need this comment? When we configure it as index it should already an intention.